sklearn/impute/_knn.py

```diff
@@ -30,11 +30,11 @@ class KNNImputer(_BaseImputer):
 
     Parameters
     ----------
-    missing_values : number, string, np.nan or None, default=`np.nan`
+    missing_values : number, string, np.nan or None, default=np.nan
         The placeholder for the missing values. All occurrences of
         `missing_values` will be imputed. For pandas' dataframes with
         nullable integer dtypes with missing values, `missing_values`
-        should be set to `np.nan`, since `pd.NA` will be converted to `np.nan`.
```

Comment:
Thanks for the PR! Why did you remove the backticks here here? I think there made sense.

Reply:
This is my first time completing a PR, thanks for the quick feedback on my contribution. 

The intent was to make the formatting consistent between that line and line 33 where the default=np.nan is referenced. 

Rejecting the suggested change on line 37 makes sense. The backticks can add clarity.